backend/package.json

```diff
@@ -5,6 +5,7 @@
   "main": "src/app.ts",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
+    "typecheck": "tsc -b --noEmit",
```

Comment:
I'm a total noob, what does `tsc` do? I'm guessing it has something to do with type security?

Reply:
This just type checks all files and tells you the errors, sort of like a linter. You can tie this to something like a pre-commit hook, but for now it's good to run semi-regularly so you don't stack up the errors and have a terrifying busy-work monster.